Sources/Controllers/2.0/Settings/ImportExport/OAExportItemsSelectionViewController.mm

```diff
@@ -320,6 +320,8 @@ - (NSString *) getTrackDescr:(NSString *)filePath appearanceInfo:(OAGpxAppearanc
     NSString *folder = @"";
     NSArray<NSString *> *pathComponents = filePath.pathComponents;
     NSString *parent = pathComponents.count > 1 ? pathComponents[pathComponents.count - 2] : @"";
+    if ([pathComponents[pathComponents.count - 3] isEqualToString:@"Documents"])
```

Comment:
Better to check if the path without the last path component equals to app.gpxPath